client/pages/components/InputTextBox.js

```diff
@@ -13,11 +14,16 @@ import store from "../../store";
  * @param {Number} rows the number of rows
  * @param {String} placeholder the placeholder to be displayed in InputTextBox
  * @param {*} text initial text in the textbox
+ * @param {boolean} isCC true if input text box is for CC otherwise false
  */
-export default function InputTextBox({ label, rows, placeholder, text }) {
+export default function InputTextBox({ label, rows, placeholder, text, isCC }) {
```

Comment:
A generalized component such as `InputTextBox` should take as generalized props as possible. I suggest this instead if you must have the `isCC` prop:
```
export default function InputTextBox({ label, rows, placeholder, text, isCC = false }) {
```
This sets the prop's default value to false, meaning it doesn't have to be updated site-wide, but you still get the new functionality you're looking for.

Reply:
Thanks for the feedback. I fixed it.